src/app/util/attribute-storage.cpp

```diff
@@ -94,6 +95,7 @@ constexpr const EmberAfAttributeMinMaxValue minMaxDefaults[] = GENERATED_MIN_MAX
 GENERATED_FUNCTION_ARRAYS
 #endif
 
+uint32_t clusterVersions[GENERATED_CLUSTER_COUNT]                   = { 0 };
```

Comment:
Shouldn't this be an array of `DataVersion`?

Reply:
done, thanks